app/Providers/RouteServiceProvider.php

```diff
@@ -19,6 +20,9 @@ public function boot(): void
     {
         $this->configureRateLimiting();
 
+        // @phan-suppress-next-line PhanTypeMismatchReturn
+        Route::bind('gtid', static fn (string $value): User => User::whereGtid($value)->firstOrFail());
```

Comment:
what does this do?

Reply:
This lets these two routes work with GTIDs automagically

https://github.com/RoboJackets/apiary/blob/32d503bff768f6a3fbbd770a5d5081cc985edf6f/routes/api.php#L76-L77

Reply:
fancy